stripe/src/resources/card.rs

```diff
@@ -37,25 +29,89 @@ impl<'a> ::serde::Serialize for CardParams<'a> {
     }
 }
 
-
+/// The resource representing a Stripe card object.
+///
+/// For more details see [https://stripe.com/docs/api#card_object](https://stripe.com/docs/api#card_object).
 #[derive(Debug, Deserialize, Serialize)]
 pub struct Card {
     pub id: String,
+    pub object: String,
+    pub account: Option<String>,
     pub address_city: Option<String>,
     pub address_country: Option<String>,
     pub address_line1: Option<String>,
-    pub address_line1_check: Option<String>, // (pass, fail, unavailable, unchecked)
+    pub address_line1_check: Option<CheckResult>,
     pub address_line2: Option<String>,
     pub address_state: Option<String>,
     pub address_zip: Option<String>,
-    pub address_zip_check: Option<String>, // (pass, fail, unavailable, unchecked)
-    pub brand: String, // (Visa, American Express, MasterCard, Discover, JCB, Diners Club, or Unknown)
+    pub address_zip_check: Option<CheckResult>,
+    pub available_payout_methods: Option<Vec<String>>,
+    pub brand: CardBrand,
     pub country: String, // eg. "US"
+    pub currency: Option<Currency>,
     pub customer: Option<String>,
-    pub cvc_check: Option<String>, // (pass, fail, unavailable, unchecked)
+    pub cvc_check: Option<CheckResult>,
+    pub default_for_currency: Option<bool>,
+    pub dynamic_last4: Option<String>,
     pub exp_month: u32,
     pub exp_year: u32,
     pub fingerprint: String,
-    pub funding: String, // (credit, debit, prepaid, unknown)
+    pub funding: CardType,
     pub last4: String,
+    pub metadata: HashMap<String, String>,
+    pub name: Option<String>,
+    pub recipient: Option<String>,
+    pub tokenization_method: Option<TokenizationMethod>,
+}
+
+#[derive(Deserialize, Serialize, PartialEq, Debug)]
+pub enum CheckResult {
+    #[serde(rename = "pass")]
+    Pass,
+    #[serde(rename = "fail")]
+    Failed,
+    #[serde(rename = "unavailable")]
+    Unavailable,
+    #[serde(rename = "unchecked")]
+    Unchecked,
+}
+
+#[derive(Deserialize, Serialize, PartialEq, Debug)]
+pub enum CardBrand {
+    #[serde(rename = "American Express")]
+    AmericanExpress,
+    #[serde(rename = "Diners Club")]
+    DinersClub,
+    #[serde(rename = "Discover")]
+    Discover,
+    #[serde(rename = "JCB")]
+    JCB,
+    #[serde(rename = "Visa")]
+    Visa,
+    #[serde(rename = "MasterCard")]
+    MasterCard,
+    #[serde(rename = "UnionPay")]
+    UnionPay,
+    #[serde(rename = "Unknown")]
+    Unknown,
+}
+
+#[derive(Deserialize, Serialize, PartialEq, Debug)]
+pub enum CardType {
+    #[serde(rename = "credit")]
+    Credit,
+    #[serde(rename = "debit")]
+    Debit,
+    #[serde(rename = "prepaid")]
+    Prepaid,
+    #[serde(rename = "unknown")]
+    Unknown,
+}
+
+#[derive(Deserialize, Serialize, PartialEq, Debug)]
+pub enum TokenizationMethod {
```

Comment:
Maybe add an `Other` variant here, in the likely case that other NFC payment methods are added?

Reply:
I am used stripe api docs for create variants this enum